message/router/middleware/poison.go

```diff
@@ -11,6 +11,9 @@ var ErrInvalidPoisonQueueTopic = errors.New("invalid poison queue topic")
 
 // ReasonForPoisonedKey is the metadata key which marks the reason (error) why the message was deemed poisoned.
 var ReasonForPoisonedKey = "reason_poisoned"
+var PoisonedTopicKey = "topic_poisoned"
```

Comment:
those keys could be made consts to prevent surprises maybe

Reply:
changed